src/server/scripts/Kalimdor/HallsOfOrigination/boss_isiset.cpp

```diff
@@ -0,0 +1,1075 @@
+
+    // Startfall / Astral Rain ability
+    SPELL_ASTRAL_RAIN_CONTROLLER            = 74381,
+    SPELL_ASTRAL_RAIN_1                     = 74134,
+    SPELL_ASTRAL_RAIN_2                     = 74365,
+    SPELL_ASTRAL_RAIN_3                     = 74371,
+
+    // Adds / Celestial Call ability
+    SPELL_ASTRAL_FAMILIAR_CONTROLLER        = 74383,
+    SPELL_CELESTIAL_CALL_1                  = 74362,
+    SPELL_CELESTIAL_CALL_2                  = 74355,
+    SPELL_CELESTIAL_CALL_3                  = 74364,
+
+    // Mana shield / Veil of Sky ability
+    SPELL_MANA_SHIELD_CONTROLLER            = 74382,
+    SPELL_VEIL_OF_SKY_1                     = 74133,
+    SPELL_VEIL_OF_SKY_2                     = 74372,
+    SPELL_VEIL_OF_SKY_3                     = 74373,
+    SPELL_VEIL_OF_SKY_DAMAGE                = 79370,
+
+    // Mirror Images (at 66% and 33%)
+    SPELL_MIRROR_IMAGE_STARRY_SKY_SPAWNER   = 69941,
+    SPELL_MIRROR_IMAGE_STARRY_SKY_N         = 69936,
+    SPELL_MIRROR_IMAGE_STARRY_SKY_E         = 69939,
+    SPELL_MIRROR_IMAGE_STARRY_SKY_W         = 69940,
+    SPELL_ASTRAL_SHIFT                      = 74312,
+    SPELL_ASTRAL_SHIFT_EXPLOSION            = 74333,
+    SPELL_MIRROR_IMAGE_SPAWNER              = 74264,
+    SPELL_MIRROR_IMAGE_N                    = 74263,
+    SPELL_MIRROR_IMAGE_E                    = 74262,
+    SPELL_MIRROR_IMAGE_W                    = 74261,
+    SPELL_ADDS_STATE_VISUAL                 = 74289,
+    SPELL_STARFALL_STATE_VISUAL             = 74265,
+    SPELL_MANA_SHIELD_STATE_VISUAL          = 74266,
+    SPELL_IMAGE_EXPLOSION                   = 74301,
+
+    // Celestial Familiar npc
+    SPELL_ORB_VISUAL                        = 74356, 
+    SPELL_ARCANE_BARRAGE                    = 74374,
+
+    // Starry Sky npc
+    SPELL_STARRY_SKY_VISUAL                 = 74149,
+
+    // Astral Shift Explosion Visual npc
+    SPELL_ASTRAL_SHIFT_EXPLOSION_VISUAL     = 74331,
+
+    // Spatial Flux & Energy Flux (heroic only)
+    SPELL_CALL_OF_SKY                       = 90750, // Summons Spatial Flux
+    SPELL_SPAWN_ENERGY_FLUX_ISISET          = 90735, // Makes random player cast Summon Energy Flux
+    SPELL_ENERGY_FLUX_BEAM_ISISET           = 90741, // Makes nearby Spatial Flux cast visual beam
+    SPELL_ENERGY_FLUX_PERIODIC              = 74044,
+};
+
+enum NPCs
+{
+    NPC_CELESTIAL_FAMILIAR                  = 39795,
+    NPC_ASTRAL_RAIN                         = 39720, // N
+    NPC_CELESTIAL_CALL                      = 39721, // E
+    NPC_VEIL_OF_SKY                         = 39722, // W
+    NPC_ISISET_SPATIAL_FLUX                 = 48707, // heroic only
+};
+
+enum Texts
+{
+    SAY_AGGRO                               = 0,
+    SAY_SUPERNOVA                           = 1,
+    SAY_SUPERNOVA_WARNING                   = 2,
+    SAY_PLAYER_KILL                         = 3,
+    SAY_DEATH                               = 4,
+};
+
+enum Events
+{
+    EVENT_NONE,
```

Comment:
i got some weird behaviors before, with enums not initialized.
i always define the first value xD